terraform/cache.tf

```diff
@@ -5,15 +5,26 @@ locals {
 resource "aws_s3_bucket" "cache" {
   provider = aws.us
   bucket   = "nix-cache"
+}
+
+resource "aws_s3_bucket_lifecycle_configuration" "cache" {
+  provider = aws.us
+  bucket   = aws_s3_bucket.cache.id
 
-  lifecycle_rule {
-    enabled = true
+  rule {
+    id     = "transition-to-standard-ia"
+    status = "Enabled"
 
     transition {
       days          = 365
       storage_class = "STANDARD_IA"
     }
   }
+}
+
+resource "aws_s3_bucket_cors_configuration" "cache" {
+  provider = aws.us
+  bucket   = aws_s3_bucket.cache.id
 
   cors_rule {
```

Comment:
Did you intend to leave `cors_rule` here?

Reply:
yes. that is the syntax

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket_cors_configuration

Reply:
Ah, sorry. I just saw "deprecated [...] `cores_rule`" in the title, and missed the important "in-line" part. Please disregard.